Makefile

```diff
@@ -84,18 +88,11 @@ fallthrough: submodules
 
 go.test.unit: $(KUBEBUILDER)
 
-# Generate manifests e.g. CRD, RBAC etc.
-manifests: vendor
+# Generate manifests e.g. CRD, RBAC etc. locally for Stacks API types
+# as it needs the custom "maxDescLen=0" option
+manifests: vendor kubebuilder.manifests
 	@$(INFO) Generating CRD manifests
-	@rm -rf cluster/charts/crossplane/templates/crds
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd:trivialVersions=true paths=./apis/cache/... output:dir=cluster/charts/crossplane/templates/crds
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd:trivialVersions=true paths=./apis/compute/... output:dir=cluster/charts/crossplane/templates/crds
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd:trivialVersions=true paths=./apis/core/... output:dir=cluster/charts/crossplane/templates/crds
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd:trivialVersions=true paths=./apis/database/... output:dir=cluster/charts/crossplane/templates/crds
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd:trivialVersions=true paths=./apis/kubernetes/... output:dir=cluster/charts/crossplane/templates/crds
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd:trivialVersions=true paths=./apis/storage/... output:dir=cluster/charts/crossplane/templates/crds
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd:trivialVersions=true paths=./apis/workload/... output:dir=cluster/charts/crossplane/templates/crds
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd:maxDescLen=0,trivialVersions=true paths=./apis/stacks/... output:dir=cluster/charts/crossplane/templates/crds
+	$(CONTROLLERGEN) crd:maxDescLen=0,trivialVersions=true paths=./apis/stacks/... output:dir=$(CRD_DIR)
```

Comment:
This keeps support of the temporary solution found for #698. :+1: 

The related issues referenced on #698 have not been resolved, so I don't think we have a better alternative for now.

Reply:
@soorena776 does this essentially generate the CRD's with full descriptions and then generate again (overwrite) them with 0 length description?

Reply:
Correct.